doc/conf.py

```diff
@@ -219,6 +217,7 @@
     'css_files': ['_static/css/custom.css'],
 }
 
+googleanalytics_id = 'UA-97947441-1'
```

Comment:
No, at least you cannot hide it...
https://webapps.stackexchange.com/questions/5440/is-it-possible-to-hide-a-google-analytics-user-id-from-source
I have also enable "hacking defense"
https://blog.kissmetrics.com/protect-analytics-from-hacking/